plone/app/search/browser.py

```diff
@@ -120,14 +140,35 @@ def types_list(self):
 
     def sort_options(self):
         """ Sorting options for search results view. """
-        return (
-            SortOption(self.request, _(u'relevance'), ''),
-            SortOption(
-                self.request, _(u'date (newest first)'),
-                'Date', reverse=True
-            ),
-            SortOption(self.request, _(u'alphabetically'), 'sortable_title'),
-        )
+        site_properties = getToolByName(self.context, "portal_properties").site_properties
+        sort_on = site_properties.getProperty('sort_on', 'relevance')
+        if sort_on == 'relevance':
```

Comment:
@rodfersou I don't understand what are you trying to do here; code on all blocks is the same.